docs/sources/howtos/bindings.md

```diff
@@ -172,6 +172,9 @@ Moreover the configuration class has a utility method `as(Class<T> configuration
 
 For example, the Yahoo Weather binding allows to configure the location and unit.
 
+### Properties
+*Things* can have properties. To retrieve the properties of a *Thing* you can call `getThing().getProperties()` inside the `ThingHandler`. The section [Properties](../architecture/thing-definition.md#Properties) on the Thing Type Definitions page provides more information about thing properties.
```

Comment:
The how-to should clearly say under which circumstance you want/need/have to add properties and guide the developer how to do it. So as stated in my other comment, the Java code examples should be moved here.